### ISSUE TITLE
Questions about the latest version

### ISSUE THREAD

**Issue by geripgeri** [dumle29/let-s-Split-v2#2]
Hi!

First of all, thanks for your work!

I would like to order the PCB via https://jlcpcb.com/ 
I don't really understand the main differences between version 2 and version 3, could you please describe them to me? Also, is the version 3 branch stabile and working? Do you recommend to use version 3?

Another question: is the version 3 branch compatible with Gateron 5 pin Switches? (Sorry for the noob question, this will be my first DIY keyboard) Can I attach led to the switches? I don't see any holes for leds, Are there any solutions for that?
Like this: ![led](https://i.imgur.com/j1fNb6n.jpg)

Thank you!

**Comment by Duckle29** [comment]
Hey there. The latest stable is onboardMCU and I actually have a batch of those on their way, so if you don't want to solder, then those are soon ready.

They, like the V3, don't support in switch LEDs though, but if you know KiCAD it shouldn't be a big change. Just get the source files, and make a footprint for sockets + leds, then update all switches to that footprint, and then re-route the traces. Should be doable in an hour or two.

I will probably be looking into leds if I make any further versions :)

**Comment by Duckle29** [comment]
Also, V3 adds the mini-usb breakout, so that your cable can come out the back of the keyboard:

https://i.imgur.com/UIVkkyn.jpg

**Comment by geripgeri** [comment]
Thank you for the response!

I think this question has been overlooked:
> Another question: is the version 3 branch compatible with Gateron 5 pin Switches?

I checked the `onboardMCU` branch, looks pretty cool :+1: I can't find the assembly docs, parts list here: https://github.com/Duckle29/lets-split-guide is this because the mentioned batch is the first?

> I actually have a batch of those on their way, so if you don't want to solder, then those are soon ready.
Could you please send me an email about the details? :slightly_smiling_face:

**Comment by Duckle29** [comment]
Yes, it's the first batch :) When you mentioned Gateron 5 pin switches, I assume you mean any MX switch with LEDs in them right? That's why I mentioned LEDs :)

**Comment by geripgeri** [comment]
> I actually have a batch of those on their way, so if you don't want to solder, then those are soon ready.

Could you please send me an email about the details? 🙂 My email address is visible on my GitHub profile.

No, I meant this: ![switch](https://ae01.alicdn.com/kf/HTB1rq.uQXXXXXciapXXq6xXFXXX0/gateron-switch-Mechanical-keyboard-cherry-clone-transparent-case-mx-brown-blue-red-switch-lighting-translucent.jpg)

**Comment by geripgeri** [comment]
Also, What is the difference between (in onboardMCU branch) `lets_split` and `lets_split_panel` folder?

**Comment by Duckle29** [comment]
Ahh those. AFAIK those are just PCB mounted switches, and yes. Those are actually preferred! :)

lets_split_panel was the first batch, where I made the PCBs but not mirrored / flippable, so I had to panelize them. Fucked up and had the switches on the bottom of the keyboard on the right-side by accident, so gave up and decided to make the mirrorable design instead.

**Comment by Duckle29** [comment]
Also, for the vitamins included batch. Expect to see it late feb :)

**Comment by geripgeri** [comment]
Thanks for the info!

Can you please write down the components of the onboardMCU? Maybe it's better if you write it to the guide repository : I would like to estimate the costs of the PCB.

**Comment by Duckle29** [comment]
I can upload the position files that I sent the Pick and place house, it has component names in there iirc.

**Comment by Duckle29** [comment]
Done https://github.com/Duckle29/let-s-Split-v2/commit/f37b36a4e8799997df2698dced4c34b87e96ebc7

**Comment by geripgeri** [comment]
Thank you!
In the meantime, there is a sale on https://jlcpcb.com 
I have a few more questions about the PCB ordering, can I please contact you directly? I don't want to spam the GitHub issue. Could you please send me an email?

**Comment by Duckle29** [comment]
@geripgeri Hey man, I don't mind the chat in here :) That way questions can help others

**Comment by thesourmango** [comment]
Nice to see you chatting here, I had pretty much the same questions.
I'm dreaming of adding a bottom row, making it 60% and making both halves wireless... #endgame?

I found a wireless lets-split called Trident (https://github.com/YCF/Trident) but it seemed to have some issues, and I'm not comfortable or familiar with the Mitosis firmware it seemed to be running.
Also no sockets.

Anyway, It's been a couple months, have any onboardMCU boards to sell? If not, have you ran into any issues with the current design?

Is there anyone working on an open source 60% wireless socketed split? Like [this](https://www.reddit.com/r/mechmarket/comments/7ywlmb/gb_40_60_and_70_wireless_split_ortholinear/) but open source (and socketed)? No offense to southpaw intended, most of us need to make a living :p

Keep up the awesome work!

**Comment by Duckle29** [comment]
@thesourmango Hey there :) The onboard MCU boards are most likely going to be called Vitamins Included, and will be sold by Novelkeys some time in April if everything goes to plan :) Mike is currently working on having cases made for them, which is one issue I've had with these boards. Due to the switches being socketed, you can't rely on them to hold the PCB in place, so the plate and the PCB have to be fixed. 

I also haven't put a diode between VCC and VBUS, which gave me some headache in firmware support, but I have since manged to solve that fully in my opinion :) Hell, in the process I even made RGB work regardless of which half is plugged in. 

As for a dual wireless, split 60%, that's very very interesting, and is in fact something I'm working on :)
Currently I'm a bit pressed for time (Job hunting, studying for EE, and looking for sponsors for a LAN event sucks the time out of the schedule :) )

If you're curious, I've been looking at using the adafruit bluefruit NRF52 based board, but I might look for cheaper options. Regardless I'll have to write a new firmware from scratch, or need a lot of help to port QMK :)

Question, Ortho or staggered? I'm leaning to staggered myself, but haven't given myself enough time to get comfortable with ortho.

**Comment by thesourmango** [comment]
Nice, I'll keep an eye out for Vitamins Included! Although I'd want to build mine from the PCB up!
I think it's awesome you're going this way, first open source, then when it works, provide a kit or a full package.

RGB? Oh yes, please upload the pcb files with pads for leds :sweat_smile:  :drooling_face: 

I don't suppose there's anything I can help you with the wireless 60% split? You haven't opened a git for it yet? :wink:  (Last year Embedded Student from Finland, ... turn it into my masters project?)

I love ortho for the looks, something magical about two squares on the desk. That being said, there are  some problems moving back and forth from an ANSI ortho to ISO Nordic Staggered when programming. :smile: So I'm now wondering if staggered would make more sense..

The mistel barocco RGB is pretty nice, the price tag isn't even that bad. [The Uruk](https://www.reddit.com/r/MechanicalKeyboards/comments/6nbvdu/ic_the_uruk_a_65_staggered_split_board/) is in development, but I'm a strong believer that the backspace is supposed to be the key in the top-right corner, so that one is out of question for me :smiley:. Also I didn't find any mention of publishing his designs.. :confused: 

On the ortho 60% QMK side, /u/SouthPawEngineer said "I will also try to have my QMK variant uploaded to Github soon.". Not a promise, but I feel like in his heart he'd want to contribute ^^

**Comment by Duckle29** [comment]
@thesourmango You can see some images of the board here :) https://imgur.com/a/sle0V

The RGB is for underglow, not for per-key RGB, but I might look in to that in the future. I have to find some backfire LEDs though, so I can keep the boards single sided load (only components on one side) as going dual-sided drives up cost. I know kaihua makes some, so I might try and get in touch there.

I'm also not quite sure if the next board will be QMK based. It might be, or it might be KiiBohd based, or custom firmware. We'll see :)

As for a repo on it, You're right. There isn't one, because I've only gotten as far as buying a bluefruit from adafruit ;)

As for the URUK, that looks sweet :)

**Comment by rockydbull** [comment]
Any chance you have a part number for the trrs socket and mini usb port? I am looking to order some blank pcbs and give these a run as macropads.

**Comment by geripgeri** [comment]
@Duckle29 Hi!

>  @thesourmango Hey there :) The onboard MCU boards are most likely going to be called Vitamins Included, and will be sold by Novelkeys some time in April if everything goes to plan :) Mike is currently working on having cases made for them, which is one issue I've had with these boards. Due to the switches being socketed, you can't rely on them to hold the PCB in place, so the plate and the PCB have to be fixed.

Based on your previous comment, is there any chance to buy the onboardMCU version from novelkeys? 

Unfortunately I only see this product, the old design. 

https://novelkeys.xyz/collections/diy-kits/products/kailh-choc-lets-split-kit

**Comment by Duckle29** [comment]
@geripgeri https://www.instagram.com/p/Bh2E833D8qm/ :)

**Comment by geripgeri** [comment]
Nice :+1:  Thank you!

**Comment by fi0** [comment]
@geripgeri not sure if this is the updated version. https://github.com/Duckle29/let-s-Split-v2/blob/f37b36a4e8799997df2698dced4c34b87e96ebc7/lets_split_panel/BOM.csv

**Comment by geripgeri** [comment]
@fi0 As far as I know that is the latest commit. Am I right @Duckle29 ?

**Comment by Duckle29** [comment]
@fi0 @geripgeri That should be the latest BOM. I haven't really don't much since the boards got made. Right now I'm just waiting for the boards to go out and be out for a while so that I can get some feedback on the design, and then I'll probably move that into a new branch called "Vitamins Included" and merge that to master.

**Comment by daysgobye** [comment]
@Duckle29 I just ordered some pcbs and trying to order other parts now. what trrs jacks does it the pcb use? from the files, it looks like smd jacks but im not sure. Any help would be appreciated. Also, thank you for the work you did!
